Casbin/Util/BuiltInFunctions.cs

```diff
@@ -128,6 +128,30 @@ public static bool KeyMatch4(string key1, string key2)
 
             return true;
         }
+        /// <summary>
+        /// KeyMatch determines whether key1 matches the pattern of key2 and ignores the parameters in key2.
+        /// For example, "/foo/bar?status=1&type=2" matches "/foo/bar"
+        /// </summary>
+        /// <param name="key1"> The first argument. </param>
+        /// <param name="key2"> The second argument. </param>
+        /// <returns></returns>
+        public static bool KeyMatch5(string key1, string key2)
+        {
+            int index = key1.IndexOf('?');
+#if NETFRAMEWORK
```

Comment:
We need not check `NETFRAMEWORK`, it will support span APIs by `System.Memory` package.